cfme/configure/access_control.py

```diff
@@ -1430,8 +1412,33 @@ def set_quota(self, **kwargs):
         view.flash.assert_success_message('Quotas for Tenant "{}" were saved'.format(self.name))
         assert view.is_displayed
 
+    def __eq__(self, other):
+        if not isinstance(other, type(self)):
+            return False
+        else:
+            return self.tree_path == other.tree_path
+
+    @property
+    def exists(self):
+        try:
+            navigate_to(self, 'Details')
+            return True
+        except CandidateNotFound:
+            return False
+
+    @property
+    def tree_path(self):
+        if self._default:
+            return [self.name]
+        else:
+            return self.parent_tenant.tree_path + [self.name]
+
+    @property
+    def parent_path(self):
+        return self.tree_path[:-1]
```

Comment:
**OPTIONAL**: Wouldn't it be nicer to use something like `self.parent.tree_path`?

Reply:
Oh yes, corrected.